Hadoop/Hadoop_Draft_one.md

```diff
@@ -0,0 +1,54 @@
+![Hadoop Art](Assets/hadoop-art.jpg)
+
+***"In pioneer days they used oxen for heavy pulling, and when one onx couldn't budge a log, they didn't try to grow a larger ox. We shouldn't be trying for bigger computers, but for more systems of computers." - Grace Hopper***
+
+In a world we are looking for results in minutes instead of hours or days instead of months, our reliance on single computer networks for data processing can be naive. In this new age, it is better to divide the workload between computers, both in terms of time and cost.
+
+If the reader has past history with computers, the idea of computer and network topologies is not new. In fact, they have been a prominent part of our network configurations (think back to Local Area Network (LAN) and Wide Area Networks (WAN) ). Although these ideas are a part of our daily life. They are limited in what the do. They provide simple network services which users use independently. But we are living in an age of Data and the amount of data flooded in the world every day is estimated, for 2020, to be around 44 zettabytes (4.4*10^22 bytes). In terms of magnitude, this is more than one disk drive for every person in the world.
+
+**Some examples of heavy data generating are:**
+
+i. The New York Stock Exchange (NYSE) generates around 4–5 terabytes of data per day. </br>
+ii. Facebook hosts more than 240 billion photos growing at 7 petabytes per month. </br>
+iii. The Internet Archive stores around 18.5 petabytes of data. </br>
+iv. The Large Hadron Collider near Geneva, Switzerland, produces about 30 petabytes of data per year. </br>

```

Comment:
not sure where it would fit in (probably _towards_ the end) but a similar section to this for which large companies/organizations use hadoop/similar would help drive home how useful these types of tools are and why the reader should care about them